pagopa/pn-frontend
# fix(PN-8874): [fe] fix on pf and pg aar to make it dowloadable after 120 days

## Short description
Fix on pf and pg aar to make it dowloadable after 120 days.

## List of changes proposed in this pull request
- Update conditions in NotificationDetail.page.tsx

## How to test
1. login as pf, open ZATG-RNKX-EXZK-202305-L-1 as Ada in TEST environment
2. Check in box "AVVISO DI AVVENUTA RICEZIONE" if the attachment is available and downloadable

_case 1_ (more than 120 days have passed): the document is not immediately downloadable, so a warning box will appear  in the "avviso di avvenuta ricezione" box (separately from the warning box in the timeline component) and the document will be dowloadable after 86400ms
_case 2_ (less than 120 days have passed): the document is immediately downloadable, so a warning box will NOT appear  in the "avviso di avvenuta ricezione" box
